src/fe/inf_fe_static.C

```diff
@@ -1267,19 +1267,28 @@ void InfFE<Dim, T_radial, T_map>::inf_compute_constraints (DofConstraints & cons
     return;
 
   if (child_elem->neighbor_ptr(0) == nullptr ||
-        child_elem->neighbor_ptr(0) != remote_elem)
-     return;
```

Comment:
Yikes, I can't believe I missed this bug.

I'm unsure why you're having to look for remote_elem at all, though.  Are we not looping over local elements only when we get here?  And if so, shouldn't we never see a remote_elem neighbor?

Reply:
I must admit that I got paranoid on `remote_elem`s recently, so I started to add checks for it everywhere. But I agree that there should be none...